README.OmniOS

```diff
@@ -1,72 +1,21 @@
-This README is new for OmniOS as of Stable release r151020 -- it is here
-because it keeps track of LX merges from OmniOS (a massive and continuing
-side-pull). LX-for-OmniOS-specific notes follow.
 
-Last illumos-joyent commit:  7f03e7512c8cd583b648bf5431882a6faaa25423
+The following commits are merged into this branch but not yet published
+to the IPS repository.
 
-LX zones can be installed using ZFS send streams (gzipped or uncompressed) from
-Joyent's images (-s /full/path/to/file), from ZFS datasets or snapshots
-(-s name/of/dataset **OR** -s name/of/this@snapshot), or from tarballs often
-used for Docker image setup (-t /full/path/to/tarball.tgz).  The
-"kernel-version" attribute MUST be set by the administrator to match the
-installed image.  For example:
+## https://github.com/omniosorg/illumos-omnios/pull/30
 
-	# zonecfg -z lx0 export
-	create -b
-	set zonepath=/zones/lx0
-	set brand=lx
-	set autoboot=false
-	set ip-type=exclusive
-	add net
-	set physical=lx0
-	end
-	add attr
-	set name=kernel-version
-	set type=string
-	set value=2.6.32
-	end
-	#
+Commits:
+ > 8381 Convert ipsec_alg_lock from mutex to rwlock
+ > 8554 Small ipsec_alginfo_t leak when zones shutdown
+ > 8529 Extended and regular SADB_ACQUIREs should share address extensio..
+ > 8541 pfiles does not properly identify PF_KEY or PF_POLICY
 
-This matches a CentOS 6.8 image from Joyent.  To install the image:
+Packages:
+ > system/kernel
+ > system/kernel/platform
+ > system/test/ostest
+ > system/extended-system-utilities
 
-     zoneadm -z lx0 install -s /root/path/to/centos68.gz
+Archive:
+ > https://downloads.omniosce.org/pkg/ipsec-backport.p5p
```

Comment:
this is just for us to keep track on the packages we have to publish later, we still have to decide if we provide archives and if so, we have to communicate and document it (e.g. on the webpage/release notes) i don't think that README.OmniOS is/should be the place for ppl to check for install info.